frontend/src/metabase/query_builder/components/DataSelector/DataSelector.jsx

```diff
@@ -1047,11 +1047,7 @@ export class UnconnectedDataSelector extends Component {
           {!isSearchActive &&
             (isPickerOpen ? (
               <SavedEntityPicker
-                collectionName={
```

Comment:
1. Virtual schema names seem broken and they do not represent collection names now
2. They were always unreliable since the collection name is not unique
3. I fix this by using `collection_id`